lib/moments/difference.rb

```diff
@@ -92,6 +92,16 @@ def in_years
       years_diff
     end
 
+    def humanized
+      diff_parts = to_hash.each_with_object([]) do |(unit, quantity), parts|
+        parts << "#{quantity} #{quantity == 1 ? unit.to_s[0..-2] : unit}" if quantity.positive?
+      end
+      last_part = diff_parts.pop
+      return last_part if diff_parts.empty?
+
+      [diff_parts.join(', '), last_part].join(' and ')
+    end
```

Comment:
I see no point in this method without I18n support.

Reply:
I agree, it is a quick hack and this was not an attempt to mimic datetime.difference_in_words, but rather a method to provide backend processes and traces a more readable version of the time lapses without the hassle of setting up translations. I can check for the presence of I18n and use the transalation if present, but introducing a dependency on I18n does not feel right either.

The author started this gem as a small and independent time calculation tool. I conceived the humanize methods in that spirit and for end-user facing time info, I will probaly rather combine it with larger and more elaborate ActiveView or equivalent libraries to get the job done. But if it helps anyone to have optional I18n support, why not?

On that topic, I have not used I18n a lot. Do you have any suggestions on how to use I18n in combining the parts with equivalents of ',' and 'and'?